asab/api/discovery.py

```
@@ -49,13 +52,57 @@ async def initialize(self, app):
 			await self.InternalAuth.initialize(app)
 
 
-	def _on_tick(self, msg):
+	def _on_tick600(self, msg):
+		# Full rescan of the advertised instances every 10 minutes
+		self.App.TaskService.schedule(self._rescan_advertised_instances())
+
+
+	def _on_zk_ready(self, msg, zkcontainer):
```

Ruff (0.14.14)

[warning] Unused method argument: `msg`

(ARG002)